atm.js

```diff
@@ -1,21 +1,107 @@
+/*
 var checkingBalance = document.querySelector("div#checking_balance");
 checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+*/
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+window.onload = function(){
```

Comment:
would recommend wrapping you're whole code into a `document.ready(function() { .. });` call